### PR TITLE
[Home] Fix for status bar inconsistency #tiny

### PR DESCRIPTION
**Pull request by sarahscott**
This fixes the finicky margin between the top of Home and the status bar.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG added.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/6af98a304f214485be35fdf735524889eecd31e4/lib/containers/artist.js">lib/containers/artist.js</a> &amp; <a href="https://github.com/artsy/emission/blob/6af98a304f214485be35fdf735524889eecd31e4/lib/containers/home.js">lib/containers/home.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: lib/containers/home.js [hunk 1]
```diff
@@ -74,8 +74,10 @@ class Home extends React.Component {
     return (
       <ListView dataSource={this.state.dataSource}
                 renderScrollComponent={(props) => {
-                  return <ScrollView {...props} refreshControl={<RefreshControl refreshing={this.state.isRefreshing}
-                                                                                onRefresh={this.handleRefresh} />} />
+                  const refreshControl = <RefreshControl refreshing={this.state.isRefreshing}
+                                                         onRefresh={this.handleRefresh} />
+                  return <ScrollView {...props} automaticallyAdjustContentInsets={false}
+                                                refreshControl={refreshControl} />
```
Comment (mennenia) [inline lib/containers/home.js]: nice one!

File: lib/containers/home.js [hunk 2]
```diff
@@ -90,7 +92,8 @@ class Home extends React.Component {
                     case 'artist':
                       return <ArtistRail ref={registerModule} key={data.__id} rail={data} />
                   }
-                }} />
+                }}
+                style={{ marginTop: 20, overflow: 'visible' }} />
```
Comment (mennenia) [inline lib/containers/home.js]: I like explicitly stating margins 👍

**Comment by mennenia** [comment]
🎉 awesome

**Pull request merged**